### PR TITLE
AppImage for Linux

### PR DESCRIPTION
**Pull request by probonopd**
Please do **not** merge yet, running into seemingly unrelated issues.

### CODE REVIEW

**Comment by drew-gross** [comment]
Looks like a build error. You may have to massage some dependencies. Also, this PR as is removes the OSX build. We'll need both OSX and linux builds working before we can merge this.

**Comment by probonopd** [comment]
Reinstated macOS, have no clue about the dependencies; this doesn't seem to be AppImage specific to me and I don't know how I can solve these.

**Comment by drew-gross** [comment]
You can use `npm run update-dependencies` to update all dependencies to the latest version. If that doesn't work, then we'll probably to put in some real effort.

**Comment by probonopd** [comment]
Ping @drew-gross

**Comment by drew-gross** [comment]
This PR has some merge conflicts and also looks like it still isn't passing tests.

File: .travis.yml [hunk 1]
```diff
@@ -1,16 +1,24 @@
...
+  - osx
+  - linux
+  
 cache:
   directories:
-  - node_modules
+    - node_modules
+    
+script:
+  - npm run update-dependencies
```
Comment (shockone) [inline .travis.yml]: We don't want to have different dependency versions than in `package.json`. It can break things.

Comment (probonopd) [inline .travis.yml]: I think it should be fine to just use the latest `electron-builder` and remove the `npm run update-dependencies`

**Comment by vshatskyi** [comment]
@probonopd I plan to close old PRs to keeps things clean. Is it OK with you if I close this one?

**Comment by probonopd** [comment]
Yes

**Pull request closed without merging**